.build/images/Quartz64/quartz64_defconfig

```diff
@@ -1122,3 +1134,16 @@ CONFIG_ZSWAP=y
 CONFIG_ZSWAP_COMPRESSOR_DEFAULT_ZSTD=y
 CONFIG_ZSWAP_DEFAULT_ON=y
 CONFIG_ZSWAP_ZPOOL_DEFAULT_ZSMALLOC=y
+CONFIG_RT_GROUP_SCHED=y
+CONFIG_INET_ESP=m
+CONFIG_INET_ESPINTCP=n
+CONFIG_INET_ESP_OFFLOAD=n
```

Comment:
Both are not defined in our kernel, so it shouldn't be required to explicitly disable them.